index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <meta 
+  name="Description"
+  content="First project creating the first two sections of my Portfolio">
+  <title>Abdel-Khafid's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div class="nav-container">
+        <div class="nav-logo">
+            <img class="my-lo" src="./assets/Abdel khafid SALAOU.png" alt="My_logo">
```

Comment:
Great job so far 👍 . As stated by the previous reviewer, the logo should be on the left and in a link a tag. This is for three reasons: convention, semantics, and user experience(UX), for further information you can check the[ link.](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)
```
<header>
 <a href="#Portfolio" class="link"><img class="my-lo" src="./assets/Abdel khafid SALAOU.png" alt="My_logo"></a>
<nav class="topnav">
</nav>
</header>

```

Reply:
Thank you @Lizdev-05 it will be done.